five-principles-that-guide-tidb-and-pingcap-II.md

```diff
@@ -0,0 +1,70 @@
+
+## Build a universal, globally-adopted database
+
+At PingCAP, our goal is to build a universal, globally-adopted database. To achieve this goal, we have adopted **Principle #4: Focus on creating a universal database**. This means:
+
+* Creating a database that could be used in any industry, in any scenario, and anywhere. We believed that TiDB should be a universal database, rather than an industry-specific one. We stuck by this idea, even though it meant at early stages we couldn't acquire customers rapidly.
+* We made sure that we had global adopters. Our adopters include companies in North America, Europe, and Asia. They've tested TiDB in a huge array of scenarios and won over many other customers.
+* We have a global workforce. Our remote workforce culture ensures that we hire the best talent, no matter where they live. We also maintain an active open source community and engage with contributors worldwide.
+* We don't ignore the "vertical" part of our product; we just focus on it when the time is right. Only when we established a full-scale, elastic, and extensible architecture, did we turn our attention to performance.
+
+## Attract top customers and the others will follow
+
+Initially, we didn't try to sell TiDB to everyone. This reflects TiDB Principle #5: **Focus on and win over high value, industry leading companies, and the smaller companies will follow**.
+
+Looking at TiDB's customer stories, our early adopters had some common characteristics, including:
+
+* Market values over 10 billion dollars.
+* The largest data volume and the most challenging scenarios.
+* Extremely high queries per second (QPS) and rigorous standards for database stability.
+
+In addition, these companies were not very sensitive to costs, so when they chose infrastructure software like databases, they weren't concerned with the price. They based their decision solely on the product itself: can the product solve their problems or not?
+
+Therefore, we decided to target TiDB at those companies, because TiDB is a distributed database designed for massive data. By verifying TiDB's usability in those large companies, we felt it would be easier to persuade smaller companies (those valued at US $1 billion) to try it. After that, if our database was proven to be stable and scalable and that it could handle various use cases, it would be even easier to win the trust of other companies.
+
+## Looking back and going forward
+
+In 2015, when we started the company, people mocked us: how dare you claim to build a new database? Just five years later, in 2020, we had adopters across the globe and closed $270 million of series D round of funding. TiDB had higher performance and lower latency, supported hybrid cloud service and elastic scaling, and became a true HTAP database. We acquired major customers in the financial and banking industries, including companies in China, Japan, and North America.
```

Comment:
You could add the blog link to "series D round of funding" and the adopter list link to "adopters."